doc/cephfs/mount-using-fuse.rst

```diff
@@ -2,57 +2,27 @@
  Mount CephFS using FUSE
 ========================
 
-Prerequisite
-------------
-Before mounting CephFS, ensure that the client host (where CephFS has to be
-mounted and used) has a copy of the Ceph configuration file (i.e.
-``ceph.conf``) and a keyring of the CephX user that has CAPS for the Ceph MDS.
-Both of these files must be present on the host where the Ceph MON resides.
+To make a choice between kernel and FUSE mounted CephFS and to go through
+the prerequisites required by both read `Mount CephFS: Prerequisites`_ page.
```

Comment:
As I mentioned in the kernel doc, let's just describe what the ceph-fuse driver here is.